autobuild/build.sh

```diff
@@ -107,12 +103,16 @@ fi
 if [[ `uname` == "Darwin" ]]; then
 	# autopoint is not in PATH after install via brew (conflicting with system gettext https://github.com/Homebrew/legacy-homebrew/issues/24070)
 	# so we can do `brew link --force gettext` or just add it to PATH before configuring which is preferable because we need it only for compiling
-	export PATH="/usr/local/opt/ccache/libexec:/usr/local/opt/gettext/bin:${PATH}"
-	export LDFLAGS="-L/usr/local/opt/gettext/lib ${LDFLAGS}"
-	export LDFLAGS="-L$(brew --prefix libomp)/lib ${LDFLAGS}"
-	export LDFLAGS="-L$(brew --prefix libtool)/lib ${LDFLAGS}"
-	export CPPFLAGS="-I/usr/local/opt/gettext/include ${CPPFLAGS}"
-	export PKG_CONFIG_PATH="/usr/local/opt/libffi/lib/pkgconfig:${PKG_CONFIG_PATH}"
+	export PATH="$(brew --prefix ccache)/libexec:$(brew --prefix gettext)/bin:${PATH}"
+	# `lib` is required to correctly detect libmng/libjpeg in autotools. Current `configure.ac` script doesn't use
+	# `pkg-config` to get library flags (libmng provides pkg-config (.pc) file from 2.0.3, but Debian still using 1.0.1).
+	export LDFLAGS="-L$(brew --prefix)/lib ${LDFLAGS}"
```

Comment:
Ok, I'm lazy enough to fix the jpeg/mng to handle pkg-config and direct lib... So, I'll approve this PR and I hope we remember to fix it there and here XD